test/DisintermediatedGrants.test.js

```diff
@@ -46,38 +45,26 @@ describe("DisintermediatedGrants", function () {
             DONATION_GRACE_PERIOD
         )
         await this.grants.deployed()
```

Comment:
`this.grants` is confusing - we should rename it

Reply:
Fixed in [github.com/researchdao/contracts/commit/c2580141307ef03af319bd238fc36569b4879c07](https://github.com/researchdao/contracts/commit/c2580141307ef03af319bd238fc36569b4879c07).